# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2007-05-07 22:46+Paris, Madrid\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"



### Added recently

#: gpicsync-GUI.py:397
msgid "Creating an 'originals-backup' folder."
msgstr ""

###

#: ..\..\..\gpicsync-GUI.py:66
msgid "Local time correction"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:67
msgid "&Options"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:69
msgid "About..."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:71
msgid "&Tools"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:72
msgid "EXIF reader"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:73
msgid "Geo-Rename pictures"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:74 ..\..\..\gpicsync-GUI.py:575
msgid "GPX Inspector"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:75
msgid "KMZ Generator"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:76
msgid "&Help"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:85
msgid "Pictures folder"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:86
msgid "GPS file (.gpx)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:87
msgid " Synchronise ! "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:88
msgid "Quit"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:89
msgid "Stop"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:90
msgid "Clear"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:91
msgid "View in Google Earth"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:93
msgid "UTC Offset="
msgstr ""

#: ..\..\..\gpicsync-GUI.py:94
msgid "Geocode picture only if time difference to nearest track point is below (seconds)="
msgstr ""

#: ..\..\..\gpicsync-GUI.py:95
msgid "Create a log file in picture folder"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:97
msgid "Dates must match"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:99
msgid "Create a Google Earth file"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:101
msgid "Google Maps export, folder URL="
msgstr ""

#: ..\..\..\gpicsync-GUI.py:103
msgid "backup pictures"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:105
msgid "interpolation"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:106
msgid "add geonames and geotagged"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:186
msgid "More informations and help:"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:203
msgid "To visualize the results in Google Earth you must either:"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:204 ..\..\..\gpicsync-GUI.py:568
msgid "- finish a synchronisation"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:216
msgid "Couldn't find or launch Google Earth"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:252
msgid "You must first select a pictures folder and a GPX file."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:264
msgid "Beginning synchronization with "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:265
msgid "UTC Offset ="
msgstr ""

#: ..\..\..\gpicsync-GUI.py:266
msgid " hours and maximum time difference = "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:274
msgid "Starting to generate a Google Earth file (doc.kml) in the picture folder ..."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:278
msgid "Starting to generate a Google Map file (doc-web.kml) in the picture folder"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:289
msgid "Geocoded with UTC Offset= "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:290
msgid " and  Maximum time difference = "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:292
msgid "Pictures Folder: "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:293
msgid "GPX file: "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:307
msgid "(Found "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:312
msgid "Processed image "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:342
msgid " (writting geonames and geotagged to keywords tag in picture EXIF)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:347
msgid "Couldn't retrieve geonames data..."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:349
msgid "FINISHED GEOCODING PROCESS"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:351
msgid "PROCESSING STOPPED BY THE USER"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:355
msgid "Adding the GPS track log to the Google Earth kml file"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:358
msgid "Click on the 'View in Google Earth' button to visualize the result"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:359
msgid "( A Google Earth doc.kml file has been created in your picture folder.)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:365
msgid "( A Google Maps doc-web.kml file has been created with the given url )"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:380
msgid "Local time corrections"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:383
msgid "Use this option ONLY if your camera local time is wrong."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:384
msgid "Indicate here the local time now displayed by your camera and GPS (hh:mm:ss)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:386
msgid "Local time displayed now by the camera"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:389
msgid "Local time displayed now by the GPS"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:392
msgid "Apply correction"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:406
msgid "EXIF Reader"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:409
msgid "Read the EXIF metadata of the selected picture."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:411 ..\..\..\gpicsync-GUI.py:412
#: ..\..\..\gpicsync-GUI.py:443
msgid "All EXIF metadata"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:413 ..\..\..\gpicsync-GUI.py:446
msgid "Date/Time/Lat./Long."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:419
msgid "Select a picture"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:441
msgid "Selected metada "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:451
msgid "lat./long."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:453
msgid "None"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:459
msgid "Renaming tool"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:462
msgid "This tool renames your pictures with the "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:462
msgid "original time/date and lat./long.(if present)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:464
msgid "Rename pictures in a folder"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:465
msgid "Rename a single picture"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:483 ..\..\..\gpicsync-GUI.py:504
msgid "Beginning renaming..."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:491 ..\..\..\gpicsync-GUI.py:518
msgid "Didn't find Original Time/Date for "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:494 ..\..\..\gpicsync-GUI.py:525
msgid "Renamed "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:508
msgid "Interrupted by the user"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:526
msgid "Finished"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:533
msgid "Create a kmz file archive"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:535
msgid "Create KMZ file !"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:543
msgid "Sorry this tool is not yet available for the Linux version"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:552
msgid "Creating a KMZ file in the pictures folder..."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:555
msgid "Adding doc.kml"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:559
msgid "Adding "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:563
msgid "KMZ file created in pictures folder"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:565
msgid "Couldn't rename the zip file to kmz (maybe a previous file already exist)"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:567
msgid "To create a Google Earth kmz file you must either:"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:569
msgid "- select a folder you've already synchronized then select the KMZ Generator tool"
msgstr ""


#: ..\..\..\gpicsync-GUI.py:578
msgid "Inspect a gpx file and show tracklog data."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:580
msgid "Select a gpx file"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:599
msgid "Select a gpx file first."
msgstr ""

#: ..\..\..\gpicsync-GUI.py:602
msgid "Looking at "
msgstr ""

#: ..\..\..\gpicsync-GUI.py:603
msgid "Number of valid track points found"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:606
msgid "Date"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:606
msgid "Time"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:607
msgid "Latitude"
msgstr ""

#: ..\..\..\gpicsync-GUI.py:607
msgid "Longitude"
msgstr ""

#: ..\..\..\gpicsync.py:186
msgid "taken "
msgstr ""

#: ..\..\..\gpicsync.py:187 ..\..\..\gpicsync.py:207
msgid "writting best latitude/longitude match to picture: "
msgstr ""

#: ..\..\..\gpicsync.py:188 ..\..\..\gpicsync.py:194 ..\..\..\gpicsync.py:208
#: ..\..\..\gpicsync.py:215
msgid " time difference (s)= "
msgstr ""

#: ..\..\..\gpicsync.py:193
msgid "no track point below the maximum time range "
msgstr ""

#: ..\..\..\gpicsync.py:193 ..\..\..\gpicsync.py:197 ..\..\..\gpicsync.py:215
msgid " : WARNING: DIDN'T GEOCODE, "
msgstr ""

#: ..\..\..\gpicsync.py:195
msgid " long="
msgstr ""

#: ..\..\..\gpicsync.py:195
msgid "For information nearest trackpoint was at lat="
msgstr ""

#: ..\..\..\gpicsync.py:197
msgid "no track point at this picture date "
msgstr ""

#: ..\..\..\gpicsync.py:210
msgid "Warning: Picture date "
msgstr ""

#: ..\..\..\gpicsync.py:211
msgid " and track point date "
msgstr ""

#: ..\..\..\gpicsync.py:211
msgid " are different ! "
msgstr ""

#: ..\..\..\gpicsync.py:215
msgid " no suitable track point below maximum time difference, "
msgstr ""

